core/src/services/dropbox/backend.rs

```diff
@@ -162,6 +166,121 @@ impl Accessor for DropboxBackend {
             _ => Err(parse_error(resp).await?),
         }
     }
+
+    async fn batch(&self, args: OpBatch) -> Result<RpBatch> {
```

Comment:
Dropbox's batch delete is not fast, maybe we don't need to implement it?

Reply:
No...If they are deleted one by one, it is easy to trigger too_many_write_operations. 😢

Reply:
And there will be a lot of requests.

Reply:
> too_many_write_operations

Ok.。。